requirements.txt

```diff
@@ -1,2 +1,5 @@
 ansible==2.8.3
 netaddr
+PyYAML==5.1.2
+aiodns==2.0.0
+aiohttp==3.6.2
```

Comment:
I would keep the web requirements in a separate file instead

Reply:
I thought about installing main requirements from web application in the background. Depends on what do you expect from that UI